### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by pborreli**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | N/A |
| License | MIT |
| Doc PR | N/A |

### CODE REVIEW

**Comment by Tobion** [comment]
master is the wrong branch, either 2.3 or 2.7

File: src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php [hunk 1]
```diff
@@ -160,7 +160,7 @@ public function write($sessionId, $data)
                 $mergeStmt->bindParam(':data', $encoded, \PDO::PARAM_STR);
                 $mergeStmt->bindValue(':time', time(), \PDO::PARAM_INT);
 
-                //Oracle has a bug that will intermitently happen if you
+                //Oracle has a bug that will intermittently happen if you
```
Comment (javiereguiluz) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php]: I know that it was wrong in the original comment too but, should't we add a white space between the `//` and the actual comment content?

Comment (pborreli) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php]: yes, I guess we can

**Comment by fabpot** [comment]
@pborreli Can you submit the PR on 2.3? Thanks.

**Pull request closed without merging**